x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchRequest.java

```diff
@@ -106,6 +115,11 @@ public EqlSearchRequest(StreamInput in) throws IOException {
         if (in.getVersion().onOrAfter(Version.V_7_10_0)) {
             resultPosition = in.readString();
         }
+        if (in.getVersion().onOrAfter(Version.V_7_12_0)) {
```

Comment:
same here

Reply:
There are a couple more instances of this test. But I assume it's fine, since there aren't bwc integration tests like we have for SQL, so this should work?